build.sh

```diff
@@ -115,35 +115,35 @@ case "$1" in
 	*) echo "ERROR: wrong command '$1'" >&2; exit 1;;
 esac
 
-if [ -z "$cmd" -o "$cmd" = "dependencies" ]; then
+if [ -z "$cmd"  ] || [ "$cmd" = "dependencies" ]; then
```

Comment:
`-o` is working on dash and busybox sh. Is there a real problem on some shell? Because shellcheck general complains aren't a real reason to change something what works (yes, shell check complains about `Prefer [ p ] || [ q ] as [ p -o q ] is not well defined.` but in this case it works).

Reply:
Strictly speaking, it's undefined behavior because it hits the case of more than 4 arguments to [

From https://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html "4 arguments: The results are unspecified. "